package.json

```diff
@@ -23,17 +23,5 @@
   },
   "eslintConfig": {
     "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
```

Comment:
@Jasper-fh I was getting errors about having a `browserslist` in both `package.json` and  in `.browserlistrc`. I chose to use the latter because it seemed more up to date. Let me know if that was the wrong thing to do.